packages/dht/src/dht/store/StoreRpcLocal.ts

```diff
@@ -37,7 +37,7 @@ interface DataStoreConfig {
 
 const logger = new Logger(module)
 
-export class DataStore implements IStoreService {
+export class StoreRpcLocal implements IStoreRpc {
```

Comment:
This class now handles more than just RpcLocal funtionality. It also includes all the "business logic" related to storing. StoreRpcLocal should only contain the functions required by `IStoreRpc`.
This same holds for many other changes in this PR such as the WebSocket and WebRTC connectors.